src/main/java/core/basesyntax/service/impl/WriteServiceImpl.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.WriteService;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Map;
+
+public class WriteServiceImpl implements WriteService {
+    @Override
+    public void writeToFile(Map<String, Integer> storage, String path) {
+        File file = new File(path);
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(file))) {
+            bufferedWriter.write(createReport(storage).toString());
+        } catch (IOException e) {
+            throw new RuntimeException("Cannot write to file" + file, e);
+        }
+    }
+
+    private StringBuilder createReport(Map<String, Integer> storage) {
+        final char comma = ',';
+        final String fruit = "fruit";
+        final String quantity = "quantity";
```

Comment:
Actually, @petro-ushchuk meant to make buffered writer a local variable, not all of these constants. Please fix the same issue in all places